features/host_can_make_offer_on_listing.feature

```diff
@@ -0,0 +1,39 @@
+Feature: Host can make an offer on a listing
+  As a host,
+  In order to make some money,
+  I want to be able to make an offer on a listing
+
+  Background:
+    Given the following listings exist
+    | pet_name  | pet_location    | pet_description   | start_date      | end_date      | pet_picture |
+    | Zane      | Gothenburg      | I'm nice          | 2019-06-28      | 2019-06-29    | picture1    |
+    | Carla     | Stockholm       | I love cats!      | 2019-05-28      | 2019-06-01    | picture2    |
+
+    When I visit the landing page
+    Then I should see "Zane"
```

Comment:
No assertions in the background

Reply:
you don't have to test this, you have already tested this in another feature that has been merged